src/components/web3/SendCryptoPopup.tsx

```diff
@@ -364,10 +379,19 @@ export const CryptoSendPopup: React.FC<CryptoSendPopupProps> = ({
               <span>Send: </span>
               <input
                 type="text"
-                pattern="[0-9]*"
-                value={value}
+                pattern="\d*.?\d*"
```

Comment:
`.` matches everything which will probably allow `NaN` through again, needs escaping with `\.`

Reply:
Fixed in [c9dbafc](https://github.com/brave/brave-talk/pull/1226/commits/c9dbafc9b7164daa55720f7d0c0c5b9c47ca4c29)